charts/ingress-nginx/values.yaml

```
@@ -127,6 +127,8 @@ controller:
     # Ingress creation gets rejected if there are multiple default IngressClasses.
     # Ref: https://kubernetes.io/docs/concepts/services-networking/ingress/#default-ingress-class
     default: false
+    # -- Annotations to be added to the ingress class
```

Just a small nitpick. Please re-run `helm-docs` once changed in values.yaml

```suggestion
    # -- Annotations to be added to the IngressClass resource.
```

---

Hi @ubergesundheit, thanks for reviewing. I added your suggested change & re-ran helm-docs. Also, squashed my commits.